### ISSUE TITLE
NTR, pulmo: Bronchial telangiectasia

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6481]
**Preferred term label:**
Bronchial telangiectasia

**Synonyms**
Endobronchial telangiectasia

**Definition (free text, please give PubMed ID)**
Telangiectasia affecting the trachoebronchial system.
Comment: Endobronchial telangiectasia may lead to hemoptysis and pulmonary hemorrhage.
•	PMID: 31466963 
•	PMID: 3359830

**Parent term (use hpo.jax.org/app)**
Mucosal telangiectasiae HP:0100579
Abnormality of the pulmonary vasculature HP:0004930

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
Remove ``Pulmonary telangiectasia`` as synonym from https://hpo.jax.org/app/browse/term/HP:0006548

**Comment by pnrobinson** [comment]
added term
